content/docs/start/index.md

```diff
@@ -1,7 +1,7 @@
 ---
-description: 'Learn how you can use DVC to manage data science and machine
-learning projects: version data, access it anywhere, capture data pipelines, and
-manage experiments.'
+description: 'Get a quick introduction to the major features of DVC for data
+science and machine learning projects: version data, access it anywhere, capture
+pipelines and metrics, and manage experiments.'
```

Comment:
Also took the liberty to update this to clarify that the Get Started is users getting the product value quick.